main.tf

```diff
@@ -341,6 +291,80 @@ resource "checkly_alert_channel" "pagerduty_ac" {
 resource "checkly_alert_channel" "slack_ac" {
   slack {
     channel = "#checkly-notifications"
-    url = "https://slack.com/webhook"
+    url     = "https://hooks.slack.com/services/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g"
+  }
+}
+
+
+resource "checkly_check_group" "test_group1" {
+  name      = "Critical User Flows"
+  activated = true
+  muted     = false
+  tags = [
+    "terraform", "critical-user-flows"
+  ]
+  concurrency = 300
+  locations = [
+    "us-west-1",
+  ]
+}
+
+resource "checkly_check" "browser-check" {
+  for_each = fileset("${path.module}/critical-user-flows", "*") // Iterates through the files in the scripts folder in your project's directory
+
+  name         = each.key // Sets the check name to match the file's 
+  group_id     = checkly_check_group.test_group1.id
+  type         = "BROWSER"
+  activated    = true
+  frequency    = 60
+  double_check = true
+  ssl_check    = false
+  locations = [
+    "us-west-1",
+    "us-east-1"
+  ]
+  use_global_alert_settings = false
+
+  alert_channel_subscription {
+    channel_id = checkly_alert_channel.slack_ac.id
+    activated  = true
+  }
+  environment_variables = { // This sets check-level environment variables
+    TEST_EMAIL      = "user@email.com",
```

Comment:
@modern-sapien do we need to duplicate `USER_EMAIL`?

Reply:
@modern-sapien I forgot to hit save on the review it seems - do these checks pass without the `PRODUCTS_NUMBER` variable being set?

Reply:
yes, the test passes regardless of the amount of items applied in the cart, which means that the expect isn't being hit.

I will update to have PRODUCTS_NUMBER included.

Reply:
I'll check after re-applying if that alters the behavior of the check.